.circleci/config.yml

```diff
@@ -229,6 +229,13 @@ jobs:
           name: Run memory tests
           command: postgrest-test-memory
           when: always
+      - run:
+          name: Run coverage
+          command: postgrest-coverage
+      - run:
+          name: Upload coverage to codecov
+          command: bash <(curl -s https://codecov.io/bash) -f coverage/codecov.json
```

Comment:
This should work without token for circleci, too! Once we're registered there, of course.

Reply:
Done! I've also added the `CODECOV_TOKEN` env var to CircleCI.

Reply:
Ok, great. Seems to work - even without the token, because "pass secrets to PRs" was not enabled. Therefore I removed the token from CircleCI again - it works without it. Less noise.